nodepool/templates/nodepool.yml.j2

```diff
@@ -324,6 +327,9 @@ providers:
         image-name: "OnMetal - Ubuntu 18.04 LTS (Bionic Beaver)"
       - name: ubuntu-xenial-onmetal
         image-name: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
+      - name: rpc-r14-trusty_loose_artifacts-swift
```

Comment:
This doesn't mirror similar images with a definition in pools[2].labels section, does that mean it's not available?